docs/pages/Nakama/plugin.mdx

```diff
@@ -0,0 +1,201 @@
+# Cardinal Plugin
+
+Go based Nakama containers can be extended using shared objects (aka [plugins](https://pkg.go.dev/plugin)). The [starter-game-template](https://github.com/Argus-Labs/starter-game-template) leverages this shared object pattern to expand on Nakama to better interact with a Cardinal based backend.
+
+The Nakama plugin bundled with the starter-game-template is general purpose, and should work with your Cardinal based game without any modifications. As you work on your game, you may find that you want to further change the Nakama plugin to better serve your specific use case. This document serves as a reference for the various aspects of the Cardinal-focused Nakama plugin.
+
+## RPC Endpoints
+
+Transactions and Reads registered with a `cardinal.World` object will automatically get Nakama RPC endpoints created for them. For example, if you set up a Transaction and a Read with the following code:
+
+```go
+var MoveTx = cardinal.NewTransactionType[MoveMsg, MoveResult]("move-a-player")
+var PositionRead = cardinal.NewReadType[PositionRequest, PositionReply]("position")
+```
+
+And register those objects with `world.RegisterTransactions` and `world.RegisterReads`, the following RPC endpoints will be created in Nakama:
+
+- /tx/game/move-a-player
+- /read/game/position
+
```

Comment:
Are these the new endpoints? I would personally still leave the old ones there as it's still usable. The new endpoints are sort of rushed while the old ones are more stable and hardened for now. Just my opinion. 

Also the /read/game/position has been replaced with /query/game/position.

Reply:
I'll hold off on submitting this PR until the new endpoints are merged.